src/ShopBundle/objects/TCMSTableEditor/TCMSTableEditorShopOrderEndPoint.class.php

```diff
@@ -105,6 +143,20 @@ public function GetFrontendActionUrlToSendOrderEmail($sMail = null)
         return $sURL;
     }
 
+    public function exportOrderToWaWi(): string
+    {
+        $translator = $this->getTranslator();
+
+        if (false !== $this->oTable->ExportOrderForWaWiHook($this->oTable->GetPaymentHandler())) {
+            $this->oTable->MarkOrderAsExportedToWaWi(true);
+
+            return $translator->trans('chameleon_system_shop.orders.export_success');
+        }
+
+        // TODO this should however be a proper error?
```

Comment:
A general should be o.k. here, as long as we log errors during the export.

Reply:
Added a simple error display